Sources/REventLogger/Networking/ConfigureUrlRequest.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+
+protocol ConfigureUrlRequest {
+    var method: HTTPMethod { get }
+    var headers: [String: String]? { get }
+    var body: Encodable? { get }
+    var path: String { get }
+    func createURLRequest(with url: URL) -> URLRequest?
+}
+
+extension ConfigureUrlRequest {
+    func createURLRequest(with url: URL) -> URLRequest? {
+        guard let url = buildUrlFrom(baseUrl: url) else {
+            return nil
+        }
+        var request = URLRequest(url: url)
+        request.httpMethod = method.rawValue
+        if let body = body {
+            do {
+                let encoder = JSONEncoder()
+                encoder.keyEncodingStrategy = .convertToSnakeCase
```

Comment:
just to understand, is this required at backend? keys will be encoded such as `app_name`, "app_version" and so on..